js_modules/dagster-ui/packages/eslint-config/rules/missing-graphql-variables-type.js

```diff
@@ -17,13 +17,12 @@ const createRule = ESLintUtils.RuleCreator((name) => name);
  *
  */
 
-const APIS = new Set(['useQuery', 'useMutation', 'useSubscription']);
-const APIToEnding = {
-  useQuery: 'Query',
-  useMutation: 'Mutation',
-  useSubscription: 'Subscription',
-  useLazyQuery: 'LazyQuery',
-};
+const apisRequiringVariableType = new Set([
+  'useQuery',
+  'useMutation',
+  'useSubscription',
+  'useLazyQuery',
+]);
```

Comment:
hmm, I could probably remove this rule and change the type definition of these to require both type parameters... we'd lose the autofixer but might be worth it over maintaining this script?

Reply:
I think it would be fine to require both, since it would be consistent, though I guess it would be a little bloaty for operations that don't require variables.

I guess it depends on how annoying the rule been to maintain, and since you've been the maintainer on it, I leave it up to you. :)

Reply:
I don't mind maintaining it. We can always change it later.